styles/colors.css

```
@@ -0,0 +1,92 @@
+  --tag-teal-text: #99f6e4;
+  --tag-yellow-bg: #78350f;
+  --tag-yellow-text: #fef3c7;
+  --tag-red-bg: #991b1b;
+  --tag-red-text: #fecaca;
+  --tag-indigo-bg: #312e81;
+  --tag-indigo-text: #c7d2fe;
+  --tag-gray-bg: #374151;
+  --tag-gray-text: #e5e7eb;
+}
+
+@custom-variant dark (&:where([data-theme='dark'], [data-theme='dark'] *));
```

Stylelint (17.5.0)

[error] Unexpected unknown at-rule "@custom-variant" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)